reactNativeProject/src/screens/Main/components/TodoList/index.js

```diff
@@ -0,0 +1,22 @@
+import TodoList from "./layout";
+import { connect } from "react-redux";
+
+import { actionCreators } from "../../../../redux/item/actions";
+
+const mapDispatchToProps = dispatch => ({
+  onRemoveItem: index => {
+    dispatch(actionCreators.onRemoveItem(index));
+  },
+
+  onToggleItemCompleted: index =>
+    dispatch(actionCreators.onToggleItemCompleted(index))
```

Comment:
Again, unify styles. In `onRemoveItem` you are defining a code block and here you are returning the dispatch directly